backend/dal/Migrations/v01.07.02/Up/PostUp/01-PropertyTypes.sql

```diff
@@ -0,0 +1,14 @@
+PRINT 'Updating PropertyTypes'
+
+-- Ensure the insert succeeds.
+DELETE FROM dbo.[PropertyTypes] where Id = 2;
```

Comment:
In our case this works fine, but not ideal. It theoretically could remove an existing property type in a database.  Generally it's best practice to only do the insert here and on down do the delete.  I wouldn't change it though, as currently we're 100% in control of the production data.